TEMPLATE.md

```diff
@@ -0,0 +1,113 @@
+---
+id: 000-template
+title: Swarm Template
+status: draft
+created: YYYY-MM-DD
+contributors:
+    -
+    -
+    -
+exit-criteria: no
+success-metrics: no
+clear-roles: no
+future-iteration: no
+roles-needed:
```

Comment:
What about additional contributors?

Reply:
@adambabik absolutely, do you think they should be in the template by default?

Reply:
added some more here